checker/jdk/index/src/java/io/File.java

```diff
@@ -195,7 +195,7 @@ int getPrefixLength() {
      * string for convenience.  This string contains a single character, namely
      * <code>{@link #separatorChar}</code>.
      */
-    public static final @MinLen(1) String separator = "" + separatorChar;
+    public static final /*!MinLen(1)*/ String separator = "" + separatorChar;
```

Comment:
We've talked about this before and I agree with you, but that change should be in a separate PR. It's on my TODO list.